docs/pages/architecture/registry.rst

```diff
@@ -0,0 +1,72 @@
+.. _nuts-registry-architecture:
+
+Nuts registry architecture
```

Comment:
forgot to mention, that is more technical than architectural documentation

Reply:
I still think this document must be under /technical/

Reply:
Moved it.